docs/development.md

```diff
@@ -123,7 +134,8 @@ centos9-katello-devel-stable:
 The contents of the specified repo will be copied to the home directory excluding the `.git` folder and `.gitignore` file.
 
 As an example, you can keep dotfiles such as `.bashrc` in the root of your git repository and install packages [with a `bootstrap` script](#running-a-boostrap-script) using this structure:
```

Comment:
I just noticed that bootstrap is misspelled as "boostrap". It's misspelled in the section below too. (Thanks copilot!)